pkg/utils/kubeclient/volume_test.go

```diff
@@ -0,0 +1,90 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubeclient
+
+import (
+	"testing"
+
+	"github.com/fluid-cloudnative/fluid/pkg/common"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+// Use fake client because of it will be maintained in the long term
+// due to https://github.com/kubernetes-sigs/controller-runtime/pull/1101
+func TestIsPersistentVolumeExist(t *testing.T) {
+
+	testPVCInputs := []*v1.PersistentVolume{&v1.PersistentVolume{
```

Comment:
A typo here. `testPVCInputs` should be `testPVInputs`

Reply:
thanks, fixed.